frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-metrics/PipelineMetricsRefreshDropdown.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import IntervalDropdown from '@console/internal/components/monitoring/poll-interval-dropdown';
+
+export interface PipelineMetricsRefreshDropdownProps {
+  interval: number;
+  setInterval: (v: number) => void;
+}
+
+const PipelineMetricsRefreshDropdown: React.FC<PipelineMetricsRefreshDropdownProps> = ({
+  interval,
+  setInterval,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <div className="form-group pipeline-metrics-dashboard__dropdown-wrap">
+      <label className="pipeline-metrics-dashboard__dropdown-title">
```

Comment:
Don't use classes that are not directly imported, makes the component dependant on another arbitrary component.